tests/tests.js

```diff
@@ -720,86 +744,210 @@ QUnit.test("validateForm test", function(assert) {
 QUnit.module( "CheckIt tests for functions that involve store", {
   beforeEach: function() {
     // prepare something before each test
+
+    localStorage.clear();
+
     this.$fixture = $('#qunit-fixture');
     this.$calendarDiv = this.$fixture.find('#calendarDiv');
+
+    // TODO add a dropdown for testing
     
     this.$fixture.append(
         `<script>
         $(document).ready( function() {
-        checkit = new CheckIt('localStorage');
+        checkit = new CheckIt('localStorage', $('#qunit-fixture #calendarDiv'));
         });
         </script>`);
+ 
     this.checkit = checkit;
```

Comment:
That's a good goal, but I'm not sure you *want* to have a checkit object that requires a fully loaded page before it can be created. Does it do page manipulation while it's being created?